### ISSUE TITLE
Problem 2.5 a

### ISSUE THREAD

**Issue by nbraynor** [ilmoi/MMLBook#5]
Perhaps you can help me with my confusion about this problem. It appears in the r2-2r1 step, your solution accidentally performed -A24-2*A14 to get a value of -3 for A24 instead of A24-2*A14 to get a 7 for A24. I don't think the correct solution is not an empty solution set. There are four equations and four unknowns and the solution is [x1 x2 x3 x4]T = [26/15 -13/30 2/5 -1/10]T. Am I thinking about this the wrong way?

**Comment by sangyun884** [comment]
I think so. A24 is 7 after subtraction

**Comment by malshoff** [comment]
No, his answer is correct. (-)2/1 = -2, so you multiply row 1 by -2 and add that to row 2. Same thing for row 3, and for row four, you'll multiply row 1 by (-5)/1 = -5, and add that to row 4. It's step 7 listed here: https://www.statlect.com/matrix-algebra/Gaussian-elimination

**Comment by raphsimon** [comment]
His answer is not correct since he doesn't even start off with the right matrix. R2 is [2 5 -7 -5 | -2] and not [2 5 -7 5| -2]. System 2.5 a does not have a solution.

**Comment by Tianmaru** [comment]
> His answer is not correct since he doesn't even start off with the right matrix. R2 is [2 5 -7 -5 | -2] and not [2 5 -7 5| -2]. System 2.5 a does not have a solution.

Thank you, I came to the same conclusion.
